packages/cli/src/lib/cmds/play.ts

```diff
@@ -0,0 +1,131 @@
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {
+  Config, GradleWrapper, JdkHelper, AndroidSdkTools, ConsoleLog, Log, GooglePlay, TwaManifest,
+} from '@bubblewrap/core';
+import {Track} from '@bubblewrap/core/dist/lib/GooglePlay';
+import * as fs from 'fs';
+import * as path from 'path';
+import {ParsedArgs} from 'minimist';
+import {TWA_MANIFEST_FILE_NAME} from '../constants';
+import {Prompt, InquirerPrompt} from '../Prompt';
+/**
+  * The Play class is the class that is used to communicate with the Google Play Store.
+  */
+class Play {
+  constructor(
+    private args: ParsedArgs,
+    private config: Config,
+    private androidSdkTools: AndroidSdkTools,
+    private googlePlay: GooglePlay,
+  ) {}
+
+  async bootstrapPlay(): Promise<void> {
+    await this.googlePlay.initPlay();
+  }
+
+  // bubblewrap play --versionCheck can validate the largest version number vs twa-manifest.json and update to give x+1 version number.
+  async getLargestVersion(): Promise<Number> {
+    // Need to get an editId, then list all apks available. This should allow us to query the highest apk number.
+    // This exists in Gradle play plugin but is not easily accessible over CLI.
+    // This should be completed in a future CL.
+    return 0;
+  }
+
+  private isInAvailableTracks(userSpecifiedTrack: string): boolean {
+    const track: string = userSpecifiedTrack.toLowerCase();
+    const selectedTrack: Track = (Track as any)[track];
+    if (selectedTrack == (null || undefined)) {
+      return false; // Should probably spit out an error message that track needs to be in [x,y,z]
+    }
+    return true;
+  }
+
+  // bubblewrap play --publish="Internal"
+  async publish(): Promise<void> {
+    // if (this.args.publish) { // This argument should be validated in run()
+    // }
+    // Validate that the publish value is listed in the available Tracks.
+    const userSelectedTrack = this.args.publish.toLowerCase() || 'internal'; // If no value was supplied with publish we make it internal.
+    if (!this.isInAvailableTracks(userSelectedTrack)) {
+      return; // Throw error message?
+    }
+    if (this.args.appBundleLocation && fs.existsSync(this.args.appBundleLocation!!)) { //appbundlelocation is an option argument.
+      await this.googlePlay.publishBundle(Track.internal, this.args.appBundleLocation);
+      return;
+    }
+    // Make tmp directory copy file over signed APK then cleanup.
+    const publishDir = fs.mkdtempSync('bubblewrap');
+    fs.copyFileSync("DEFAULT_FILE_PATH", path.join(publishDir, "DEFAULT_FILE_NAME")); // Need help on default file path and default file name.
+    await this.googlePlay.publishBundle(Track.internal, publishDir);
+    
+    fs.rmdirSync(publishDir);
+  }
+
+  private validServiceAccountJsonFile(path: string | undefined): boolean { // Return an error or boolean? Log a message?
```

Comment:
We do have an Error type - https://github.com/GoogleChromeLabs/bubblewrap/blob/main/packages/core/src/lib/Result.ts